.github/workflows/main.yml

```diff
@@ -41,12 +35,57 @@ jobs:
       env:
           TZ: Europe/Zurich
           CI: true
+
+  build-dev:
+    name: Build lib in dev mode
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
@subotic Do you happen to know if there could be one central version matrix? This would be easier to update.

Reply:
You could try and see if [defaults](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaults) could allow you to define a value which you then could use in all the different matrix statements.

Reply:
will do, thx